corehq/apps/hqadmin/management/utils.py

```diff
@@ -100,7 +100,7 @@ def _get_prs_by_label(pr_infos):
         for label in pr['labels']:
             if label['name'] in LABELS_TO_EXPAND:
                 prs_by_label[label['name']].append(pr)
-    return prs_by_label
+    return dict(prs_by_label)
```

Comment:
what's the need for this? It's already a dict

Reply:
Django templates can't iterate over a defaultdict (we display this in the email), so had to switch to a normal dict

Reply:
interesting, didn't know that